app/controllers/declaration/AdditionalActorsSummaryController.scala

```diff
@@ -58,7 +59,7 @@ class AdditionalActorsSummaryController @Inject()(
         validYesNo =>
           validYesNo.answer match {
             case YesNoAnswers.yes => navigator.continueTo(mode, controllers.declaration.routes.AdditionalActorsAddController.displayPage)
-            case YesNoAnswers.no  => navigator.continueTo(mode, nextPage)
+            case YesNoAnswers.no  => navigator.continueTo(mode, controllers.declaration.routes.AuthorisationProcedureCodeChoiceController.displayPage)
```

Comment:
Nit: `controllers.declaration.` can be removed. Personally I would use this as a general rule for a better readability